addons/istio/istio.yaml

```
@@ -26,7 +26,7 @@ spec:
     - matchLabels:
         kubeaddons.mesosphere.io/name: prometheus
   kubernetes:
-    minSupportedVersion: v1.16.0
+    minSupportedVersion: v1.17.0
```

so we can only deploy this addon on konvoy 1.5.x and newer. But I guess since its preview it should be okay.

---

why we have to bump the `minSupportedVersion`?

---

I think I did this errantly, fixing
```suggestion
    minSupportedVersion: v1.16.0
```